docs/source/notebooks/04_Emulating_hydrological_models.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```

Comment:
<p>by adding the <code>workdir</code> path... will be made available from the PAVICS Jupyter environment. The default <code>workdir</code> setting puts model outputs in the temporary directory <code>/tmp</code> , which is not visible from the file Jupyter file explorer. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/raven/pull/424/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Good precision, fixed.</p>